Game.java

```diff
@@ -64,29 +59,15 @@ public void assignPlayers() {
 
     public void playGame(Board board) {
         assignPlayers();
-        print(gameMode.toUpperCase() + " game is starting..");
-        print("Where would you like to go? (1, 2, 3, 4, 5, 6, 7, 8, 9)");
-        while (!board.checkForDraw() || !board.getState().equals("Game Over")) {
+        consoleGame.drawNewGame();
+        while (!board.isGameOver()){
             board = playerMakesMove();
```

Comment:
you pass in an argument of _board_ to playGame which supercedes the global _board_, and then you re-assign _board_ to the result of _playerMakesMove_, but you don't pass it to _playerMakesMove_, rather you use the _board_ global variable.
This is trouble.  What if your argument  changed to _gameBoard_ and all of the references to board in _playeGame_ changed too.  Then in playerMakesMove() you would have the wrong _board_.

This is a small code base, but could get you in trouble in a large legacy system!